### PR TITLE
adding results.ipynb

### PR DESCRIPTION
**Pull request by vinayakSharm**
adding notebook to analyze the ddb results

### CODE REVIEW

File: benchmarking/video_quality/results.ipynb [hunk 1]
```diff
@@ -0,0 +1,1297 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def get_step(video_location):\n",
```
Comment (mg6955) [inline benchmarking/video_quality/results.ipynb:227]: A Comment /Markdown on what get_step() does

Comment (vinayakSharm) [reply benchmarking/video_quality/results.ipynb:92]: done!

File: benchmarking/video_quality/results.ipynb [hunk 2]
```diff
@@ -0,0 +1,1053 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Find the original yolo and media pip scores\n",
```
Comment (mg6955) [inline benchmarking/video_quality/results.ipynb:244]: media pipe -typo

Comment (vinayakSharm) [reply benchmarking/video_quality/results.ipynb]: fixed

**Review by mg6955 — APPROVED** [review]

**Pull request merged**